src/config/index.js

```diff
@@ -29,6 +29,8 @@ const configuration = {
   INFO_EMAIL: process.env.INFO_EMAIL || '',
   ZENDESK_KEY: process.env.ZENDESK_KEY,
   ZENDESK_LOGO_URL: process.env.ZENDESK_LOGO_URL,
+  POPULAR_PRODUCTS: process.env.POPULAR_PRODUCTS,
+  TRENDING_PRODUCTS: process.env.TRENDING_PRODUCTS,
```

Comment:
Default value is missing. Also move the along with General recommendations to a separate section above `Miscellaneous`.
```suggestion
  // Recommendation constants
  GENERAL_RECOMMENDATIONS: process.env.GENERAL_RECOMMENDATIONS || '[]',
  POPULAR_PRODUCTS: process.env.POPULAR_PRODUCTS || '[]', 
  TRENDING_PRODUCTS: process.env.TRENDING_PRODUCTS || '[]',
```

Reply:
Got it. Although, this case is handled within component, I will remove it from there.

Reply:
Addressed